core/src/main/java/info/rmapproject/core/utils/Terms.java

```diff
@@ -112,6 +112,9 @@
 
  	/** The term for the tombstoned status. */
  	public static final String RMAP_TOMBSTONED = "tombstoned";
+ 	
+    /** Progenitor DiSCO as lineage ID */
+ 	public static final String RMAP_LINEAGE_PROGENITOR = "progenitor";
```

Comment:
Noting the interchangeable use of lineage, progenitor, and lineageProgenitor... unless @emetsger or @birkland can see any reason not to change this - rmap:lineageProgenitor seems more accurate - I like it!